### ISSUE TITLE
Disney Epic Mickey 2: The Power of Two (245300) 

### ISSUE THREAD

**Issue by lividhen** [ValveSoftware/Proton#6670]
# Compatibility Report
- Name of the game with compatibility issues: Epic Mickey 2: The Power of Two
- Steam AppID of the game: 245300

## System Information
- GPU: AMD custom GPU 0405 
- Driver/LLVM version: mesa 22.2.0
- Kernel version: 5.13.0
- Link to full system information report as [Gist](https://gist.github.com/): [gist](https://gist.github.com/lividhen/2f9b1905a5f8d4b7fd8064d38012af56)
- Proton version: 7.0-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-245300.log](https://github.com/ValveSoftware/Proton/files/11180505/steam-245300.log)

## Symptoms 
Menu and ui are green. 

## Reproduction
Open game using latest proton. (currently 7.0-6)

**Comment by alasky17** [comment]
@lividhen Unfortunately, I think you might have some issue with your system.  Are you using a Steam deck?  It looks like you have a VR headset plugged in... I would suggest trying to unplug extra peripherals, and if that doesn't work, try reinstalling the game.

**Comment by lividhen** [comment]
Yes I am on a deck. No peripherals are plugged in and I have reinstalled the game, but the issue persists.

**Comment by alasky17** [comment]
Are you running SteamOS Stable?  Could you upload the current full system information report?

**Comment by lividhen** [comment]
I am on SteamOS Stable. Here is my system report from the steam help menu. https://gist.github.com/lividhen/2f9b1905a5f8d4b7fd8064d38012af56

**Comment by alasky17** [comment]
@lividhen Could you look up what settings you have in the launcher for the game?  If you have set Steam to automatically skip the launcher, you will need to go to the game properties by clicking the gear icon -> Properties -> General and scroll down to "Launch Options" and then change that back to "Ask when starting game" or "Run the Launcher".  

Then in the launcher, please list your display mode, resolution, if v-sync and anti-aliasing are on or off, etc.

**Comment by lividhen** [comment]
Display Mode: Borderless Window
Resolution: 1280x800
Vsync: Off
Anti-aliasing: On
Dynamic Shadow Quality: High
Ground Shadow Quality: High

**Comment by alasky17** [comment]
@lividhen In your earlier report, you had mentioned that the game worked well on 7.0-1 and broke sometime in between - is that accurate?  If so, we can attempt to confirm if it is indeed a regression.  Weirdly, I still cannot reproduce the issue on my deck even with your exact settings, which makes this difficult to troubleshoot.

**Comment by lividhen** [comment]
The previous issue for this game said the issue was resolved on 7.0-1 but I have not been able to test as I can't get it to build from source. It could be that I have a messed up driver somewhere.

**Comment by kisak-valve** [comment]
Hello @lividhen, as a blind guess, have you experimented with a SteamVR driver like ALVR on your Steam Deck in the past? I suspect that is true and that there's either an Oculus Rift S wirelessly connected or that ALVR always reports the HMDs presence, even if there is a wireless interruption.

This might be unrelated to the issue you're seeing with the game.

**Comment by lividhen** [comment]
Yes I have experimented with vr. There is no active connection, but there may be some persistent profile from alvr. I have attempted to remove usb devices inside steam vr but it tells me to run it as administrator. Running as root causes steam vr not to open.

**Comment by alasky17** [comment]
> The previous issue for this game said the issue was resolved on 7.0-1 but I have not been able to test as I can't get it to build from source. It could be that I have a messed up driver somewhere.

Luckily, it is easy for me to create a build so that you can test this :)  I created a build that is basically identical to 7.0-1, except that it can be tested more easily by:

1) Dropping the tar bundle in ~/.steam/steam/compatibilitytools.d (you might have to create this folder if it does not exist already).
2) Untar the bundle.
3) Restart Steam/the steam deck.
4) Proton 7.0-1 should show up as an option to select from the available Proton options :) 

I uploaded the tar here with lock code "proton": https://www.codeweavers.com/xfer/alasky/proton-7.0/

If you still are seeing the green video with that build, I agree that it is likely that something on your system got messed up.

**Comment by lividhen** [comment]
Still green with that version of proton. Had to re-image my system because I broke something, and the issue still occurrs.

**Comment by alasky17** [comment]
Very strange.  It also seems like re-imaging your system would have fixed anything that was going wrong ... could you get a picture of the problem, and a log with +quartz,+strmbase,+strmbase_qc,+gstreamer,+amstream,+qcap,+qcap_v4l,+qedit,+mciqtz,+mci,+mfplat,+gstreamer,+evr,+dxva2,+wvmcore added in addition to the default logging channels?

**Comment by lividhen** [comment]
Apologies, but are those to be added to the launch options?

**Comment by alasky17** [comment]
No worries - If I'm remembering correctly, you can use PROTON_LOG=+xyz,+abc %command% to add those instead of =1 :)

**Comment by lividhen** [comment]
[steam-245300.log](https://github.com/ValveSoftware/Proton/files/11259085/steam-245300.log)   


<img src="https://user-images.githubusercontent.com/42221797/232704451-329d2763-8636-471e-b357-76b7623f7ba4.jpg" width="500">
<img src="https://user-images.githubusercontent.com/42221797/232704478-3c37c847-9655-4db1-8c2c-3e6c663ab86f.jpg" width="500">

**Comment by GenericGaming** [comment]
If I may interject, trying to get em2 to function has been a personal project of mine for a while, and ive found someone online who claims to have gotten the game working through the use of a tool named "dgvoodoo". I've been experimenting with dgvoodoo but have had no luck so far, maybe that could be the answer?

**Comment by GenericGaming** [comment]
just confirmed that the experimental proton 7.0-1 does NOT work

**Comment by pka-human** [comment]
Hello. Was there any solution to the problem? Has anyone managed to launch it without problems with the green interface?

**Comment by lividhen** [comment]
Have not found any way to get around it except pulling out my old Wii 😓

**Comment by LizzyFleckenstein03** [comment]
The game breaks when there are dots in its path, i.e. when it's located in the default steam library at ~/.local/share/Steam

The bug should be fixed in Proton soon, but for now a possible workaround is creating a different Steam library that does not have dots in the path and moving the game there.

**Comment by kisak-valve** [comment]
Hello @LizzyFleckenstein03, based on what you wrote, I'd try setting the game's launch options to `PROTON_SET_GAME_DRIVE=1 %command%` and see what it does.

**Comment by LizzyFleckenstein03** [comment]
@kisak-valve yes, that also works

**Comment by pka-human** [comment]
> Hello @LizzyFleckenstein03, based on what you wrote, I'd try setting the game's launch options to PROTON_SET_GAME_DRIVE=1 %command% and see what it does.

It works, thanks!
